service/version_bundles.go

```diff
@@ -0,0 +1,78 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func newVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "kubernetes",
+					Description: "enable encryption at rest",
+					Kind:        versionbundle.KindAdded,
+				},
+				{
+					Component:   "kubernetes",
+					Description: "update to version 1.9.0",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "kubernetes",
+					Description: "use vanilla (previously coreos) hyperkube image",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "calico",
+					Description: "update to version 3.0.1",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "calico",
+					Description: "remove calico-ipip-pinger",
+					Kind:        versionbundle.KindRemoved,
+				},
+				{
+					Component:   "calico",
+					Description: "remove calico-node-controller",
```

Comment:
The description was defined as a sentence so far. I would like to keep it that way. See for instance https://github.com/giantswarm/kvm-operator/blob/master/service/version_bundles.go.

Reply:
Would you also like to use past form?
I'd like to to keep imperative form so we follow "standard", but I can live with either.

Reply:
I actually don't mind. I only want to have one convention that we use everywhere. I started with just something. When there is something better lets agree on it and get it straight everywhere. AKA business as usual.

Reply:
@marians probably has a strong opinion on this.

Reply:
everything that we have here is related to k8scc, so  imo "source of thruth" should be there.

Can we move this piece to k8scc?

Reply:
I'd be fine to keep `[]versionbundle.Changelog` & `[]versionbundle.Component` instance for every version in k8scloudconfig repo. But we'd have to merge it with operator specific stuff. Like OS and docker version. But there is `append` :)

I'd like to hear @xh3b4sd opinion on that.

Reply:
Doesn't make sense to me. It is good to have it hard coded in the operator. That makes us thing. We always have to think about updates and its implications. We would also only redirect for a portion of the changelogs, but not for all. Then everything would be different. The consistency we have right now is way more valuable IMO.

Reply:
Portion of changelogs is an argument for me.
IMO consistency is the main problem here. For example v3.0.0 now switched to coredns. It's very easy to forget about updating version_bundles.go on next `dep ensure -update` run.

Reply:
When updating vendor deps will ever be an issue it means somebody did not do his job right, which can be a problem for everything. When you add something to k8scloudconfig you have to make sure it lands in the operators accordingly. When we version everything and somebody adds something new then nothing changes for the running guest clusters when updating vendor deps so this is not an issue at all. When you screw the process though and update existing templates and you do not fix this in the operators and modify the version bundles accordingly then you fucked up greatly. Nobody should do this anyway. Something like this should not pass any review.

Reply:
Regarding the description text: As we will present this to end users without any change, please use these rules:

- past tense
- Uppercase first letter
- Punctuation only required if you use more than one sentence.

This would turn `remove calico-node-controller` into `Removed calico-node-controller`